### ISSUE TITLE
input placeholder should be disabled if the original input is disabled

### ISSUE THREAD

**Issue by knightq** [fdeschenes/bootstrap-tag#7]
If I have:

```
<input data-provide="tag" data-source="[\"a\",\"b\",\"c\"]" value="a,b" disabled="disabled"></input>
```

The right expected behaviour would be:
- tag visible and well formatted (as span)
- tag not deletable
- new tags not insertable (the placeholder input added by bootstrap tag as next sibling of the input above should be disabled too).

Does it sounds?

In other words, tags modifiability should depend on original input modifiability.

**Comment by knightq** [comment]
Here the fiddle: http://jsfiddle.net/J6AKv/8/

**Comment by fdeschenes** [comment]
I agree. Good catch! Somehow, the thought of disabling the input never crossed my mind. I'll come up with a fix for this shortly.
